snapshots/who/2023-04-03/flu_elderly.py

```diff
@@ -0,0 +1,36 @@
+"""Script to create a snapshot of dataset 'Influenza Vaccination Coverage (WHO; UNICEF, 2023)'."""
```

Comment:
Maybe some additional instructions on how to get the Excel file are needed, in case someone else updates this.

Reply:
Good point! Will add